.claude/commands/x-coverage.md

```
@@ -0,0 +1,35 @@
+# PHP Code Coverage Analysis
+
+Collect code coverage data for the specified PHP script or test suite.
+
+## Usage
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)